agent/handlers/task_server_setup_test.go

```diff
@@ -1720,6 +1782,9 @@ func TestTaskHTTPEndpointErrorCode404(t *testing.T) {
 		"/v4/v3-endpoint-id/task/",
 		"/v4/v3-endpoint-id/task/stats/",
 		"/v4/v3-endpoint-id/task/stats/wrong-path",
+		"/graphql",
+		"/graphql/v3-endpoint-id/?query=%7B__schema%7Btypes%7Bname%7D%7D%7D",
```

Comment:
I was thinking it's probably better to express this as follows

```go
"/graphql/v3-endpoint-id/?query="+url.QueryEscape("{__schema{types{name}}}"),
```

see: https://play.golang.org/p/o2KSuU-bjxn